_component_design/time-picker.md

```diff
@@ -0,0 +1,186 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Specifications
+
+### Default state
+
+- Tabbing to a time picker will set focus on the time picker but not trigger the menu.
+- Keyboard input or clicking anywhere on the element triggers the drop-down menu.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+<embed src="{{site.cdn_url}}/assets/images/components/inputs-and-controls/time-picker/time-picker-default-state.svg"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Focused and expanded
+
+#### Predictive behavior
+
+- When the user enters a number on the keypad, the menu will jump to that number.
+- For example, if the user types “7”, 7:00 p.m. will be scrolled to the top of the menu.
+
+#### Default time shown
+
+- The default times shown in the drop-down menu start at 12:00 p.m.
+- If specific times are available, the earliest one would show first, chronologically followed by later times.
+
+#### Default number of visible items
+
+- The number of items visible in the menu is four, unless the use case requires otherwise
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+<embed src="{{site.cdn_url}}/assets/images/components/inputs-and-controls/time-picker/time-picker-focused-and-expanded-states.svg"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Invalid states
+
+- If an invalid entry occurs, stack the error icon and message underneath the help text.
+- When the user replaces the invalid entry with a valid one, hide the error message and icon.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+<embed src="{{site.cdn_url}}/assets/images/components/inputs-and-controls/time-picker/time-picker-invalid-states.svg"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Idle and disabled
+
+- After the user has selected the time and shifted focus away, the time picker adopts its default state.
+- When disabled, the contents of the picker cannot be modified or triggered. Consider adding help text describing why the picker is disabled.
```

Comment:
When disabled, the user cannot modify or trigger the contents of the picker.

Reply:
Done.